python/libucx/pyproject.toml

```diff
@@ -33,6 +34,17 @@ libucx = "libucx"
 [project.urls]
 Homepage = "https://github.com/openucx/ucx"
 
+[tool.rapids-build-backend]
+build-backend = "setuptools.build_meta"
+commit-files = [
+    "libucx/GIT_COMMIT"
+]
+dependencies-file = ""
```

Comment:
This worked! The backend issued the expected warning, but didn't fail.

```text
  /pyenv/versions/3.11.9/lib/python3.11/contextlib.py:137: UserWarning: File not found: ''. If you want rapids-build-backend to consider dependencies from a dependencies file, supply an existing file via config setting 'dependencies-file'.
```

https://github.com/rapidsai/ucx-wheels/actions/runs/9356772541/job/25755034182?pr=9#step:7:6302

Reply:
Cool that seems fine to me. I don't want to introduce dependencies.yaml here. We don't need to maintain any dependencies except those for the build backend.

Reply:
alright yeah great, thank you!